yarn.lock

```diff
@@ -2430,6 +2430,14 @@
     "@jridgewell/resolve-uri" "3.1.0"
     "@jridgewell/sourcemap-codec" "1.4.14"
 
+"@kaizen/a11y@^1.7.7":
```

Comment:
We definitely don't want this change either. This probably showed up because the package was considered deleted from the repo, but is used as a dep for some of the other packages.